.changeset/mighty-carpets-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/components": patch
+---
+
+Add stateLabels back to theme-preval.
```

Comment:
We can delete this changeset because we never shipped `theme-preval` without `stateLabels`. So we didn't actually _change_ anything yet 😉